style.css

```diff
@@ -322,3 +322,62 @@ body {
 .contact-submit {
   margin-top: 1rem;
 }
+
+@media screen and (min-width: 992px) {
+  .headline {
```

Comment:
- [ ] Kudos on using a good `min-width` for the desktop version 🔥.
- [ ] There is a white space at the bottom of your page on the desktop version. Please let's remove the white space so that your app looks professional and matches the Figma template.

Your page:
<img width="686" alt="Loom_jEQblk8W2N" src="https://user-images.githubusercontent.com/81468675/149230436-7cd3ac42-ee27-4c3f-b52e-d52437696965.png">